frontend/public/components/modals/delete-pvc-modal.tsx

```diff
@@ -42,21 +44,24 @@ const DeletePVCModal = withHandlePromise<DeletePVCModalProps>((props) => {
   return (
     <form onSubmit={submit} className="modal-content">
       <ModalTitle>
-        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Delete Persistent Volume Claim
+        <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+        {t('public~Delete PersistentVolumeClaim')}
       </ModalTitle>
       <ModalBody>
         <Stack hasGutter>
           {alertComponents}
           <StackItem>
-            Are you sure you want to delete <strong className="co-break-word">{pvcName}</strong>{' '}
-            Persistent Volume Claim?
+            <Trans t={t} ns="public">
+              Are you sure you want to delete <strong className="co-break-word">{pvcName}</strong>{' '}
```

Comment:
I think this needs to be

```suggestion
              Are you sure you want to delete <strong className="co-break-word">{{pvcName}}</strong>{' '}
```

https://react.i18next.com/latest/trans-component

Reply:
updated